macros/models/_sector/tokens/transfers_base.sql

```diff
@@ -0,0 +1,65 @@
+{% macro transfers_base(blockchain, traces, transactions, erc20_transfers) %}
+{%- set token_standard_20 = 'bep20' if blockchain == 'bnb' else 'erc20' -%}
+{# denormalized tables are not yet in use #}
+{%- set denormalized = True if blockchain in ['base'] else False -%}
+
+WITH transfers AS (
+    SELECT block_time
+    , block_number
+    , tx_hash
+    , value AS amount_raw
+    , CAST(NULL AS varbinary) AS contract_address
```

Comment:
can we use `{{VAR("ETH_ERC20_ADDRESS"}}` here instead of `NULL`. :thinking:

Reply:
this is a bit confusing to me tbh, sometimes I see `0xeeee...` used as this as well. and would `0x0000...` be consistent across chains?

Reply:
yes that's why we need to start using the variable, we can probably rename it to `NATIVE_ERC20_ADDRESS` and use it across chains. 
We can change the variable to `0xeeee..` or `0xee` (or even `null` I guess) if we want.

Reply:
relevant: this [draft EIP-7528](https://eips.ethereum.org/EIPS/eip-7528)

Reply:
Updated this to use a native contract address. I will update this to use a variable in a separate PR since we need to change the name there and edit a few models after.